lib/algebra.ts

```diff
@@ -87,11 +87,11 @@ export interface BaseOperation
 export interface Paths extends BaseOperation
 {
     type : types.PATHS;
+    start:  IriTerm ;
+    via: IriTerm;
+    end: IriTerm ;
```

Comment:
Check my previous review, and also look through the PATHS docs again; start and end must be variables. Another property is required to hold the optional IRIs.